apps/src/music/blockly/blocks/simple2.js

```diff
@@ -39,6 +40,24 @@ export class GeneratorHelpersSimple2 {
     ${functionImplementationsCode}
   `;
   }
+
+  // Return a function name in JavaScript.
+  // Adapted from Blockly.JavaScript.nameDB_.safeName_
+  // at https://github.com/google/blockly/blob/498766b930287ab8ef86accf95e9453018997461/core/names.ts
```

Comment:
I took a look and I believe this is necessary unless Blockly makes `Blockly.JavaScript.nameDB_.safeName_` a public function.  In Music Lab, Blockly is generating calls to the functions with the appropriate safe names.  However, we manually generate those function implementations because we are wrapping their bodies with sequencing functionality, and we are also sometimes generating our own when_run implementation which calls those functions, and in both cases, we want to use the appropriate safe names as well.

Reply:
Makes sense. Thanks for looking into it some more!